doxylicious/README.md

```diff
@@ -0,0 +1,229 @@
+# Doxylicious
```

Comment:
I see what your doing with the name, but I'm not a fan. Maybe `kroxylicious-executable-docs`, but I think this is likely to be useful across multiple projects which have adopted modular documentation so I think we need to make this independently consumable (in the fulness of time)

Reply:
I agree the name is not great. But it's difficult to capture what it does in something succinct. It's not exactly executable docs, because "docs" encompass more than just procedures. 

I also agree that this would be better factored out into something reusable in the longer term (assuming we find this approach works well enough for us).